lib/forest_liana/engine.rb

```diff
@@ -74,7 +74,7 @@ def eager_load_active_record_descendants app
           if ForestLiana.env_secret || ForestLiana.secret_key
             unless rake?
               bootstraper = Bootstraper.new
-              bootstraper.synchronize unless ENV['FOREST_DEACTIVATE_AUTOMATIC_APIMAP']
+              bootstraper.synchronize unless ENV['FOREST_DEACTIVATE_AUTOMATIC_APIMAP'] || ENV['FOREST_DISABLE_AUTO_SCHEMA_APPLY']
```

Comment:
As discussed this morning we don't want 2 environment variables doing the same job.

Reply:
So do we have to rename the variable on express ? The old environment name is confusing, we don't know if we disable the forestadmin-schema.json generation or if it's just the sending of the apimap

Reply:
I think we should keep the new environment because of that

Reply:
Okey I'll do that